libs/feature/record/src/lib/data-view-table/data-view-table.component.ts

```diff
@@ -70,7 +70,10 @@ export class DataViewTableComponent {
   constructor(private mdViewFacade: MdViewFacade) {}
 
   fetchData(link: MetadataLinkValid): Observable<{ id: string | number }[]> {
-    if (link.protocol.startsWith('OGC:WFS')) {
+    if (
```

Comment:
Check use for the second time here => mutualise in a helper if possible.
Notice that if it was already the case, you would have to modify the code once and don't care about where it's used, now you need to change the test everywhere you use it.